apps/web/app/(app)/environments/[environmentId]/settings/(organization)/members/components/DeleteOrganization.tsx

```diff
@@ -41,6 +43,15 @@ export const DeleteOrganization = ({
     setIsDeleting(false);
   };
 
+  const deleteDisabledWarning = useMemo(() => {
+    if (!isMultiOrgEnabled) return "Organization deletion is disabled on your instance of Formbricks";
```

Comment:
This is only the UI check. Using the action `deleteOrganizationAction` deleting the organization is still possible. Please make sure to add a separate check to the `deleteOrganizationAction`.